### ISSUE TITLE
Unique ID required on iFrame for Safari

### ISSUE THREAD

**Issue by philBrown** [nyroDev/nyroModal#35]
Hi

This is more of a heads-up. Safari has this weird issue where it will cache an iFrame based on its ID. When you don't set an ID, Safari treats all iFrames as the same.

I've only seen this issue when working with iFrames in lightbox-style situations however it is definitely a real concern.

The solution is to generate a unique ID when building the iFrame, for example

```
var iframe = $('<iframe>').attr('id', 'nyromodal-iframe-' + new Date().getTime());
```

Cheers

**Comment by nyroDev** [comment]
Add unique ID for iframes. Closed by fc10b7972724d7378385194eeb5dbbd786391a6e

**Comment by nyroDev** [comment]
Thanks for your report and your patch!
